cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,45 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generateUser');
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('should send the form with filled fields and check modal', () => {
+    const {
+      userName,
+      email,
+      mobile,
+      subject,
+      currentAddress,
+      state,
+      city
+    } = generateUser();
+
+    cy.findById('firstName').type(userName);
+    cy.findById('lastName').type(userName);
+    cy.findById('userEmail').type(email);
+    cy.findById('userNumber').type(mobile);
+    cy.findById('currentAddress').type(currentAddress);
+    cy.findById('state').type(state);
+    cy.findById('city').type(city);
```

Comment:
The `findById` custom command is used here, but it seems to be incorrectly implemented. The `findById` command should be defined to select elements by their ID, but the IDs used here (e.g., 'firstName', 'lastName', etc.) should be verified to ensure they match the actual IDs in the HTML. Make sure the IDs are correct or adjust the command accordingly.